app/unitPages.rb

```diff
@@ -174,6 +170,7 @@ def getIssue(id)
 
 
 def unitSearch(params, unit)
+  # ToDo: remove this section now that unit search display looks just like global search
```

Comment:
Well, we can refactor at some point. For now I'd say at least update the comment to make clear what the code is currently used for.